### PR TITLE
update Index.html and style.css

### PR DESCRIPTION
**Pull request by Nahid1911**
Toolbar and headline section for review

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @Nahid1911 

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious


## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good pull request description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.menu-button {
+  display: flex;
+  justify-content: end;
+  margin-right: 10%;
+}
```
Comment (braincee) [inline style.css:14]: Good job implementing the CSS Flexbox as it is part of the project requirements. 👍

File: style.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+.main-container {
+  background-image: url(media/header-illsutration-mobile.svg);
+  height: 700px;
+}
```
Comment (braincee) [inline style.css:19]: - [ ] Please I highly recommend using CSS Flexbox for the `main-container` class as it is part of the project requirements. 😄

File: style.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+.main-text {
+  margin: 0% 8% 10% 8%;
```
Comment (braincee) [inline style.css:22]: - [ ] Good job implementing responsive values in your CSS rules. 😄

File: style.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;700&display=swap');
```
Comment (braincee) [inline style.css:1]: - [ ] Please I highly recommend sticking to the [Figma Design Template] per project requirements. 😄 

_**REQUIRED FIGMA DESIGN TEMPLATE**_

Screenshot (96)


_**YOUR DESIGN**_

Screenshot (94)

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+          don’t hesitate to contact me.</p>
+        </div>
+
+        <ul class="social-media">
+         <li><a><img src="media/twitter.svg" alt="twiter lonk"></a></li>
+         <li><a></a><img src="media/Linkedin icon.svg" alt="linkedin link"></a></li>
+         <li><a><img src="media/medium.svg" alt="medium link"></a></li>
+         <li><a><img src="media/github.svg" alt="github link"></a></li>
+         <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
+        </ul>
```
Comment (braincee) [inline Index.html:34]: Good job implementing the required changes by the previous reviewer. 👍

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Nahid1911  ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅

REQUIRED CHANGES ♻️
------

- [ ] Please I suggest adding responsive breakpoints per the project requirements. 😄 
       The minimum mobile screen size should be `375px`
       The minimum desktop screen size should be `768px`

- [ ] Please I suggest adding a meaningful title and short summary for the pull request. You can also use bulletins for your summary so the pull request looks professional. Here is a [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to the professional repository's basic rules. 😄 
- [ ] Please I suggest adding [HTML & CSS Linters](https://github.com/microverseinc/linters-config/tree/master/html-css) per the project requirements. 😄 
- [ ] Please I highly recommend adding a README file to the `fix_toolbar_and_header` feature branch. I can find the README file in the `main` branch, please make it a necessity to add it to the `fix_toolbar_and_header` feature branch per general requirements. 😄 

**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Nahid1911) [reply style.css:1]: @braincee , good to see your review, i did that but as per the previous reviewer's comment, I have removed that portion. Now, you are asking to do it again, i do not know why?

Comment (Nahid1911) [reply style.css:1]: @braincee , for your information
image

File: style.css (hunk 2 above)
Comment (Nahid1911) [reply style.css:19]: @braincee required change made.
image

**Comment by Nahid1911** [comment]
All the possible required changes have been incorporated, but some requests are irrelevant, I did not expect this type of poor review, it consumed my review.

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Nahid1911, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (braincee) [reply style.css:1]: Please your design is to fix the mobile version screen. In addition to the design, you are supposed to add `MY LOGO` at the top left corner of the navigation bar

**Pull request merged**